client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -107,7 +111,7 @@ export class RegistrantExtraInfoCaForm extends React.PureComponent {
 		}
 
 		// Set the lang to FR if user languages is French, otherwise leave EN
-		if ( this.props.userWpcomLang.match( /^fr-?/i ) ) {
+		if ( this.props.userWpcomLang && this.props.userWpcomLang.match( /^fr-?/i ) ) {
```

Comment:
Super minor but this could use optional chaining like `if ( this.props.userWpcomLang?.match( /^fr-?/i ) )`

Reply:
Definitely makes the code cleaner, I've made this change.